src/script/auth/component/EntropyCanvas.tsx

```diff
@@ -1,7 +1,9 @@
-import React, {MouseEvent, useRef, useEffect, useState} from 'react';
+import {CSSObject} from '@emotion/react';
+import {MouseEvent, useRef, useEffect, useState} from 'react';
 import {usePausableInterval} from '../../hooks/usePausableInterval';
 
 interface CanvasProps {
+  css?: CSSObject;
```

Comment:
With React18 we need to be explicit with every single attribute of our components (`children` needs to be explicitly set now) (see https://reactjs.org/blog/2022/03/08/react-18-upgrade-guide.html#updates-to-typescript-definitions)